ui/dialog.js

```diff
@@ -433,10 +435,11 @@ $.widget( "ui.dialog", {
 	},
 
 	_title: function( title ) {
-		if ( !this.options.title ) {
+		if ( this.options.title ) {
+			title.text( this.options.title );
+		} else {
 			title.html( "&#160;" );
 		}
-		title.text( this.options.title );
```

Comment:
I would say the same of every change here. If nothing else, specifically counting on `undefined` seems risky.